contrib/exporters/core/pipeline.go

```diff
@@ -121,6 +128,9 @@ func init() {
 	StorerHandlers = make(HandlersMap)
 	StorerHandlers.Register("stdout", NewStoreStdout, true)
 	StorerHandlers.Register("s3", NewStoreS3, false)
+
+	AccRejHandlers = make(HandlersMap)
+	AccRejHandlers.Register("none", NewAccRejNone, true)
```

Comment:
I think you forgot to register also the real action handler

Reply:
It is registered in the main.go of vpclogs